weave/index/index_pandas.py

```diff
@@ -142,7 +144,7 @@ def to_pandas_df(self, max_rows=1000, **kwargs):
         """
 
         self._sync_if_needed()
-        return self.index_df.head(max_rows)
+        return self.index_df.iloc[offset:offset+max_rows]
```

Comment:
wouldn't having the offset+max_rows potentially give you a value higher than all the rows in the index? does that cause an error trying to pull more rows than there are?

Reply:
That doesn't create an error in Python slicing. If the start index is higher than the list length, it will return an empty list. If the end index is higher it will return up to the last element.

Reply:
Oh good to know